public/app/features/logs/components/LogRowMessage.tsx

```diff
@@ -58,14 +58,18 @@ const LogMessage = ({ hasAnsi, entry, highlights, styles }: LogMessageProps) =>
   return <>{entry}</>;
 };
 
-const restructureLog = (line: string, prettifyLogMessage: boolean): string => {
+const restructureLog = (line: string, prettifyLogMessage: boolean, wrapLogMessage: boolean): string => {
   if (prettifyLogMessage) {
     try {
       return JSON.stringify(JSON.parse(line), undefined, 2);
     } catch (error) {
       return line;
     }
   }
+  // With wrapping disabled, we also want to turn it into a single-line log entry
```

Comment:
`// With wrapping disabled, we want to turn it into a single-line log entry unless the line is expanded. `

Changing the comment in this way makes more sense.

Reply:
Great suggestion, thank you.